src/H5Dmpio.c

```diff
@@ -273,6 +373,185 @@ static int H5D__cmp_filtered_collective_io_info_entry_owner(const void *filtered
 /* Local Variables */
 /*******************/
 
+#ifdef H5Dmpio_DEBUG
```

Comment:
The below code adds debugging to H5Dmpio similar to that in the MPI I/O file driver.